src/Estimators/src/LeggedOdometry.cpp

```diff
@@ -385,7 +387,8 @@ iDynTree::FrameIndex LeggedOdometry::Impl::getLastActiveContact(const std::map<i
         }
     }
 
-    double latestTime{std::numeric_limits<double>::max()}; // assuming time cannot be negative
+    std::chrono::nanoseconds latestTime{std::chrono::nanoseconds::max()}; // assuming time cannot be
```

Comment:
I tried to replicate the same logic implemented before. Here we have `std::chrono::nanoseconds::max()` since we had  `std::numeric_limits<double>::max()`. while in 

https://github.com/ami-iit/bipedal-locomotion-framework/blob/060520ccff61f563030fd5a3c31de2dd0fb52080/src/Estimators/src/LeggedOdometry.cpp#L349

we have `std::chrono::nanoseconds::min()` because before we had {-1}. I can try to modify https://github.com/ami-iit/bipedal-locomotion-framework/blob/060520ccff61f563030fd5a3c31de2dd0fb52080/src/Estimators/src/LeggedOdometry.cpp#L349 with `std::chrono::nanoseconds{-1}`

Reply:
As you prefer. Were those values meant to specifically make some searches fail? Maybe in one case, it was meant to be the first occurrence of some event, while the other was the last occurrence?